docs/dev/setup/aeolus.rst

```diff
@@ -0,0 +1,83 @@
+.. _Aeolus Setup:
+
+Aeolus Setup
+------------
+
+This section describes how to set up the external service Aeolus for custom build plans for Jenkins and Integrated code lifecycle, in Artemis.
+
+If you are setting Artemis up for the first time, these are the steps you should follow:
+
+- Install and run Docker: https://docs.docker.com/get-docker
+- :ref:`Start Aeolus`
+- :ref:`configure artemis for aeolus`
+- :ref:`Start Artemis with Aeolus`
+
+.. _Start Aeolus:
+
+Start Aeolus
+^^^^^^^^^^^^
+
+Start Aeolus with the following docker compose command:
+
+.. code-block:: bash
+
+   docker compose -f docker/aeolus.yml up -d
+
+
+Check if Aeolus is running by visiting the following URL in your browser: http://localhost:8090/docs which should
+display the Aeolus API documentation.
+
+.. _configure artemis for aeolus:
+
+Configure Artemis to use Aeolus
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Append the following lines to your file ``src/main/resources/config/application-local.yml``:
+
+.. code-block:: yaml
+
+       aeolus:
+           url: http://localhost:8090
+
+
+The server URL is the URL of the Aeolus server. The default port is 8090. And is enough for Aeolus to work with Artemis in a local environment.
+
+API Key
++++++++
+
+Without an API key, Artemis will send the location and credentials of the CI system with each request to Aeolus, which is fine
+for a local environment. However, in a production environment, it is recommended to use an API key to authenticate the requests.
```

Comment:
I don’t understand how setting an access restriction to the Aeolus API avoids having Artemis to send the Jenkins credentials.
I think what you want is two different ideas that got mixed up here:
- authenticate the communication between Aeolus and Artemis to make sure *only* Artemis can create build plans in the CI system via Aeolus
- when using Artemis with an external CI system (i.e. Jenkins), Aeolus has to communicate with this external system directly and needs the necessary login credentials to do so.

Reply:
Okay, since the API key and CI authentication are actually coupled, this makes sense then.